bipedalwalker_eng.rst

```diff
@@ -0,0 +1,191 @@
+BipedalWalker
+~~~~~~~~~~~~~~~
+
+Overview
+==========
+
+In the BipedalWalker environment, the agent needs to output 4-dimensional continuous actions to control the 2D biped robot to move forward on rough terrain. Applying motor torque at each step will get a negative reward, and each step forward will get a positive reward. If you successfully move to the farthest point, you can get more than 300 points reward. If the robot falls on the way, it will get -100 points and the game will end. The state of the agent is a 24-dimensional continuous vector, including hull angle speed, angular velocity, horizontal velocity, vertical velocity, joint position and joint angular velocity, marks that the leg touches the ground, and 10 lidar rangefinder measurements . Note that the state vector does not contain the coordinates of the robot.
+
+.. image:: ./images/bipedal_walker.gif
+   :align : center
+
+Install
+========
+
+Installation Method
+--------------------
+
+Just install the gym and box2d libraries, which can be installed by pip or combined with DI-engine
+
+.. code:: shell
+
+# Method1: Install Directly
+pip install gym
+pip install box2d
+# Method2: Install with DI-engine requirements
+cd DI-engine
+pip install ".[common_env]"
+
+Verify Installation
+-------------------
+
+After the installation is complete, you can verify that the installation was successful by running the following command on the Python command line:
+
+.. code:: python
+
+import gym
+env = gym.make ('BipedalWalker-v3')
+obs = env.reset ()
+print( obs.shape ) # (24,)
+
+Mirror

```

Comment:
Use `Image` rather than `Mirror`, and also remind other companions

Reply:
Done